app/models/bot/slack.rb

```diff
@@ -117,7 +117,7 @@ def get_project(model)
 
   def get_project_for_annotation(model)
     p = nil
-    p = model&.annotated&.project if ['ProjectMedia', 'ProjectSource'].include?(model.annotated_type)
+    p = model&.annotated&.project if model.annotated_type == 'ProjectMedia'
```

Comment:
We want to remove `project` from all Slack notifications. So this method can go away completely.

Reply:
Yes will do this when remove `project_id`

Reply:
sounds good thanks